sqe-database-access/SignInterpretationRepository.cs

```diff
@@ -132,5 +145,99 @@ UserInfo user
 				return returnSignInterpretation;
 			}
 		}
+
+		public async Task UpdateSignInterpretationCharacterById(
+				UserInfo user
+				, uint   signInterpretationId
+				, string character
+				, byte   priority
+				, uint   attributeValueId)
+		{
+			using (var transactionScope =
+					new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
+			using (var conn = OpenConnection())
```

Comment:
We should most definitely figure out a way to not open multiple connections per request, this really messes things up with locking. We need a very good performance reason to have more than one connection per request. I'm not sure how we're going to do that, but we really should.